tests/conftest.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 
 sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+sys.path.append(os.path.join(os.path.dirname(__file__), 'utils'))
```

Comment:
Open to different approaches here, but I wanted to separate the test helpers from the tests.